packages/server/test/e2e/1_browserify_babel_es201_spec.coffee

```diff
@@ -10,11 +10,37 @@ describe "e2e browserify, babel, es2015", ->
     e2e.exec(@, {
       spec: "browserify_babel_es2015_passing_spec.coffee"
       snapshot: true
+      noTypeScript: true
     })
 
   it "fails", ->
     e2e.exec(@, {
       spec: "browserify_babel_es2015_failing_spec.js"
       snapshot: true
       expectedExitCode: 1
+      noTypeScript: true
     })
+
+describe "e2e typescript", ->
```

Comment:
I think these should be in their own e2e spec file. This file was specifically for addressing an issue regarding babel and es2105. TypeScript support probably deserves its own file (e.g. `1_typescript_support_spec.ts`)